src/views/Home/Assignment/index.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { useState, useEffect, useCallback, useMemo } from 'react';
+
+import { isDefined } from '@togglecorp/fujs';
+
+
+import ListView from '#rsu/../v2/View/ListView';
+import FormattedDate from '#rscv/FormattedDate';
+import Icon from '#rscg/Icon';
+import Pager from '#rscv/Pager';
+import Button from '#rsca/Button';
+
+import useRequest from '#utils/request';
+import { notifyOnFailure } from '#utils/requestNotify';
+
+import {
+    ProjectElement,
+    MultiResponse,
+} from '#typings';
+
+import _ts from '#ts';
+
+import styles from './styles.scss';
+
+interface Assignment {
+    id: number;
+    createdAt: string;
+    projectDetails: ProjectElement;
+    createdByDetails: {
+        id: number;
+        displayName: string;
+        email: string;
+    };
+    contentObjectDetails: {
+        id: number;
+        title: string;
+    };
+    isDone: boolean;
+    contentObjectType: {
+        id: number;
+        title: string;
+    };
+}
+
+interface Data {
+    key: number;
+    assignee: string;
+    sourceTitle: string;
+    sourceURL: string;
+    projectName: string;
+    date: string;
+}
+
+interface Props {
+}
+
+interface AssignmentRendererProps extends Assignment {
+    handleClick: (id: number) => void;
+}
+
+const emptyLink = '#';
+const maxItemsPerPage = 5;
+
+function assignmentRenderer(info: AssignmentRendererProps) {
```

Comment:
Component should be capitalized: AssignmentRenderer
```suggestion
function AssignmentRenderer(info: AssignmentRendererProps) {
```